src/views/Contacts.vue

```diff
@@ -152,7 +151,7 @@ export default {
 							{
 								icon: 'icon-download',
 								text: 'Download',
-								action: this.downloadGroup(group)
+								action: this.downloadGroup(group)		// does not work if called via 'self'. why is that?
```

Comment:
You need to redefine self to this before the return declaration (line 139) :wink: 
`const self = this`

Reply:
Also, you can add comments in github (like I'm doing right now) instead of in the code, it's easier to discuss. :hugs:

Reply:
`const self = this` was not the problem, I did that. My problem was that I had a function returning a function so I could not use a function as a reference. But now I can. Solved.